app/assets/javascripts/discourse/app/components/reviewable-item.js

```diff
@@ -418,6 +402,33 @@ export default class ReviewableItem extends Component {
     }
   }
 
+  async _claimReviewable() {
```

Comment:
```suggestion
  async #claimReviewable() {
```

Minor nitpick but using the `#` prefix will allow Javascript to enforce the privacy of this method.